lib/kitchen/directions/bake_theorem/v1.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Kitchen::Directions::BakeTheorem
+  class V1
+    def bake(theorem:, number:)
+      theorem['use-subtitle'] = true
+      new_subtitle = theorem.title.cut
+
+      theorem.replace_children(with: %(<div class="os-note-body">#{theorem.children}</div>))
+      note_body = theorem.first('.os-note-body')
+
+      note_body.prepend(sibling:
+        <<~HTML
+          <div class="os-title">
+            <span class="os-title-label">#{I18n.t(:theorem)} </span>
+            <span class="os-number">#{number}</span>
+            <span class="os-divider"> </span>
+          </div>
+        HTML
+      )
```

Comment:
make a card/issue to change occurrences of this to `PrependNumberedTitle.v1(in: note_body, title: :theorem, number: number)`